java/runtime/src/main/java/io/ray/runtime/gcs/GlobalStateAccessor.java

```diff
@@ -0,0 +1,48 @@
+package io.ray.runtime.gcs;
+
+import com.google.common.base.Preconditions;
+import java.util.List;
+
+
+public class GlobalStateAccessor {
+  private long globalStateAccessorNativePtr = 0;
+
+  public GlobalStateAccessor(String redisAddress, String redisPassword) {
+    globalStateAccessorNativePtr = nativeCreateGlobalStateAccessor(redisAddress, redisPassword);
+    Preconditions.checkState(globalStateAccessorNativePtr != 0,
+        "Global state accessor native pointr is non-nullptr");
+  }
+
+  public boolean connect() {
+    return this.nativeConnect(globalStateAccessorNativePtr);
+  }
+
+  public void disconnect() {
+    this.nativeDisConnect(globalStateAccessorNativePtr);
+  }
+
+  public List<byte[]> getAllJobInfo() {
```

Comment:
Document these public methods. Especially, we need to explain that the return value is a list of protobuf-serialized `GcsJobInfo` objects.